examples/13-iamserviceaccounts.yaml

```diff
@@ -18,6 +18,11 @@ iam:
       labels: {aws-usage: "application"}
     attachPolicyARNs:
     - "arn:aws:iam::aws:policy/AmazonS3ReadOnlyAccess"
+  - metadata:
+      name: aws-load-balancer-controller
+      namespace: kube-system
+    wellKnownPolicies:
```

Comment:
would `attachWellKnownPolicies` be better?

Reply:
I think it's a _little_ odd that this is would be a verb, when the general idea of declarative config suggests using nouns instead, to describe the resulting state, not how you get there.

`attachedWellKnownPolicies` perhaps, but that's pretty verbose, and only subtly different, so likely to lead to muscle-memory mistakes.

I think the various uses of `attach` in eksctl config stand out as verbs in a sea of nouns, but I guess using `attach` specifically is consistent here, e.g., we also have `securityGroups.attachIDs`, not `securityGroups.attachedIDs`.

Reply:
Yes, I was trying to avoid a verb but maybe `withWellKnownPolicies`?

Reply:
Would I be at risk of ruining everything by saying I am not a fan of `WellKnown`? `withCommonPolicies`?

Reply:
the one problem with "common" is that it has two meanings, one "well known", the other "shared"...

Reply:
yeh that's totally right. `withNamedPolicies`? although I suppose all policies have names.

Reply:
Struggling to find a better alternative to `WellKnownPolicies` so I'm happy to stick with that. I'm still unsure about `with` vs `attach`. I know that using the word `attach`  isn't technically correct, but IMO its worth using it to remain consistent with the other fields.